spec/models/role_spec.rb

```diff
@@ -5,10 +5,19 @@
   let!(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
   let!(:cfp_role) { Role.find_by(name: 'cfp', resource: conference) }
   let!(:organizer) { create(:user, role_ids: organizer_role.id) }
+  let(:user) { create(:user) }
 
   it 'get_users' do
     expect(organizer_role.users).to include organizer
     expect(organizer_role.users.count).to eq 1
     expect(cfp_role.users).to eq []
   end
+
+  it 'adds and removes user' do
```

Comment:
We are actually testing something else here: 'does not delete role when last user is removed' (or sth along those lines)